### PR TITLE
New WDL to create VAT tsvs from previously generated BigQuery table.

### PR DESCRIPTION
**Pull request by gbggrant**

### CODE REVIEW

File: scripts/variantstore/variant_annotations_table/GvsCreateVATFilesFromBigQuery.wdl [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+
+        echo_date "concatenating $files"
+        cat $(echo $files) > vat_complete.tsv.gz
+        echo_date "bgzipping concatenated file"
+        cat vat_complete.tsv.gz | gunzip | bgzip > vat_complete.bgz.tsv.gz
+        echo_date "copying bgzipped file to ~{output_path}"
+        gsutil -m cp vat_complete.bgz.tsv.gz ~{output_path}
```
Comment (mcovarr) [inline scripts/variantstore/variant_annotations_table/GvsCreateVATFilesFromBigQuery.wdl:243]:
```suggestion
        gcloud storage cp vat_complete.bgz.tsv.gz ~{output_path}
```

File: scripts/variantstore/variant_annotations_table/GvsCreateVATFilesFromBigQuery.wdl [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+
+        echo_date "concatenating $files"
+        cat $(echo $files) > vat_complete.tsv.gz
```
Comment (mcovarr) [inline scripts/variantstore/variant_annotations_table/GvsCreateVATFilesFromBigQuery.wdl:239]: this is still magical to me 🪄

**Review by mcovarr — APPROVED** [review]
LGTM despite my seagulling 👍

**Comment by gbggrant** [comment]
Successful run of GvsCreateVATFilesFromBigQuery here: https://app.terra.bio/#workspaces/gvs-dev/GVS%20Quickstart%20v3%20ggrant/job_history/71d521c4-2034-45d0-aaac-3eb404c5fdc9

File: scripts/variantstore/variant_annotations_table/GvsCreateVATFilesFromBigQuery.wdl (hunk 2 above)
Comment (gbggrant) [reply scripts/variantstore/variant_annotations_table/GvsCreateVATFilesFromBigQuery.wdl:239]: Yeah, same. I just copied it. 🤷

File: scripts/variantstore/variant_annotations_table/GvsCreateVATFilesFromBigQuery.wdl [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+
+        # note: tab delimiter and compression creates tsv.gz files
+        bq query --nouse_legacy_sql --project_id=~{project_id} \
+        'EXPORT DATA OPTIONS(
+        uri="~{export_path}",
+        format="CSV",
+        compression="GZIP",
+        overwrite=true,
+        header=false,
+        field_delimiter=" ") AS
```
[the remark this answers is not in the document]
Comment (gbggrant) [reply scripts/variantstore/variant_annotations_table/GvsCreateVATFilesFromBigQuery.wdl:71]: "Error in query string: Error processing job 'gvs-
internal:bqjob_r171d52c52c0d6fcf_00000185cbfc3198_1': DREMEL only supports
single character delimiters. Cannot use "tab"

(possibly can use '\t'?

**Pull request merged**